### PR TITLE
Removing Rococo Testnet from the code and builds

### LINKED ISSUE #1899: Remove Frequency Rococo
Blocked Pending Frequency Rococo shutdown

Remove Frequency Rococo from all remaining places in the code and release setup

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to remove all references to the old Rococo Testnet.

Closes #1899

# Discussion

## How to test
Search for rococo, and find (almost) nothing! (Cargo lock will still mention it for now)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,13 +7,10 @@ all: build
 clean:
 	cargo clean
 
-.PHONY: start, start-relay, start-frequency, start-frequency-docker, start-manual, start-interval, start-interval-short, start-with-offchain, start-frequency-with-offchain, start-manual-with-offchain, start-interval-with-offchain
```
Comment (wilwade) [inline Makefile:10]: I think this was my fault early on, but Makefile `.PHONY` is space separated not `, ` separated

File: scripts/js/onboard/authUpgrade.js
Comment (wilwade) [inline scripts/js/onboard/authUpgrade.js:1]: Also formatted

File: scripts/js/onboard/enactUpgrade.js
Comment (wilwade) [inline scripts/js/onboard/enactUpgrade.js:1]: Also formatted

File: tools/ci/release-notes/release-notes.md.tera [hunk 2]
```diff
@@ -4,12 +4,11 @@ Full changelog: https://github.com/frequency-chain/frequency/compare/{{env.PREVI
 
 ## Versions
 - Rust Toolchain: `{{env.RUST_TOOLCHAIN}}`
-- Substrate/Polkadot: `v{{env.POLKADOT_VERSION}}`
+- Polkadot-SDK: `v{{env.POLKADOT_VERSION}}`
```
Comment (wilwade) [inline tools/ci/release-notes/release-notes.md.tera:7]: Corrected name

**Review by enddynayn — APPROVED** [review]
👍 great!

**Review by JoeCap08055 — APPROVED** [review]
lgtm
Built locally & ran tests

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**